get_ior_results.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+if [ $# -eq 0 ] ; then
+    echo "Usage: $0 <full path to results directory>"
+    exit 1
+fi
+
+RES_DIR=$1
+
+if [[ ! -d "$RES_DIR" ]] ;  then
+    echo "Result directory $RES_DIR does not exist"
+    exit 1
+fi
+
+cd $RES_DIR
+
+#Test name
+current=${PWD##*/}
+echo $current
+
+#Output file
+result="$RES_DIR/result_$current"
+
+# Extract results from test logs
+
+echo Num_Servers,Targets,Clients,Ranks,Write-Easy,Read-Easy > $result
+
+for i in *
+do
+    if [ -d "$i" ] && [[ "$i" = log* ]]; then
+       if [ -f "$i"/ior* ] && grep "Max Write" "$i"/ior* ; then
```

Comment:
I think you need to handle the situation where a test hangs/fails and you end up having no/incomplete output.
Here, it will likely just skip it and no indication of failure in the table for the specific configuration.

Reply:
True, here it just skips and gives the good case results.  There are two ways to handle the bad cases. One is to add the checks to these scripts and flag the failing cases in a separate failed tests file for each. The second will be to have a separate script to examine the failed cases and flag them.  In the second approach, we could add logic to collect log information for the failed cases.  Overall, I prefer the second approach and intend to do that when my tests are added to the new test framework.

Reply:
Yes, I think it will be great to report out test pass/fail status per testcase/variant.
Example: 
Testcase Scale_Server_Constant_Client: PASS
Testcase Scale_ServerToClient_1to4 : FAIL
    Test Variant Server128_Client32: FAIL

And I think you can implement the above when you integrate the tests into the framework.
However, I think for this result parsing scripts in this PR, you still need to log an entry in the table for tests that failed. Perhaps default values to 0?
Else, the axis scale will be incorrect when you pull it in excel.

Reply:
The way it is it will skip a missing config and not plot that server value. So the plot will show up. However, you have a point in that week to week comparison will be affected if in future we add weekly results to one sheet to compare.